infra/charts/turing/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: "Turing: ML Experimentation System"
 name: turing
-version: 0.1.7
+version: 0.1.8
```

Comment:
Why do you need to change the helm chart version?

Reply:
the migration scripts are packaged with the helm chart, so if its not bumped, there might be some issues with caching and stuff.